bridge/client/app/_components/ktb-sequence-controls/ktb-sequence-controls.component.scss

```diff
@@ -0,0 +1,2 @@
+@import '../../../variables';
```

Comment:
Is this needed when no styles are added? Otherwise, I think you can also delete the whole file.

Reply:
I'll delete the file, because it is not used.